### PR TITLE
Feature/pong

### PR DESCRIPTION
**Pull request by mercurrie**
completed features for pong game

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import time
+from turtle import Screen
+from paddle import Paddle
+from ball import Ball
+from scoreboard import Scoreboard
+
+direction = 10
```
Comment (michaelhousman) [inline main.py]: same comments as the last PR.  Lots of hard-coded values that I'd shift to the top of the file and, ideally, in a JSON.

**Review by michaelhousman — CHANGES_REQUESTED** [review]
Same comments as the last PR

File: main.py (hunk 1 above)
Comment (mercurrie) [reply main.py]: converted variables into constants and placed at top of each file.

**Review by michaelhousman — APPROVED** [review]
Looks good

**Pull request merged**